apps/data-pipeline/library-traffic-scraper/src/lib.ts

```diff
@@ -129,24 +132,24 @@ export async function doScrape(DB_URL: string) {
   const db = database(DB_URL);
   const lookup = await collectLocationMeta();
 
-  const now = new Date();
-  const minutes = now.getMinutes();
+  const dateNow = new Date();
+  const minutes = dateNow.getMinutes();
 
   for (const [id, meta] of Object.entries(lookup)) {
-    const status = getScrapeStatus(
-      meta.libraryName === "Langson Library"
-        ? "LL"
-        : meta.libraryName === "Science Library"
-          ? "SL"
-          : "LGSC",
-    );
+    const libraryCodeMap: Record<string, "LL" | "SL" | "LGSC"> = {
```

Comment:
you can simplify this type hint by casting the object literal to `const`, giving you maximal type information